### PR TITLE
[gtkmm] update version to 4.9.2

### LINKED ISSUE #29348: [gtkmm] update to 4.8.0
Library name: GTKmm
Library name: GTK
New version number:  4.8.0

Other information that may be useful (release notes, etc...)
Source is updated in official site https://download.gnome.org/sources/gtkmm/4.8/
https://download.gnome.org/sources/gtk/4.8/

### PR DESCRIPTION
**Pull request by FrankXie05**
Fixes #29348 

Update `gtkmm` to v4.9.2.
No feature needs to test.
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [ ] ~~Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by FrankXie05** [comment]
Remove Option `install-tests`.
```
..\src\7031045a84-9bcf1b2f74.clean\meson.build:1:0: ERROR: Unknown options: "install-tests"
```
https://gitlab.gnome.org/GNOME/gtk/-/commit/3121f88265ac61733e257f5335473d6f139f158c

File: ports/gtk/portfile.cmake [hunk 1]
```diff
@@ -11,6 +11,12 @@ vcpkg_from_gitlab(
         0004-macos-build.patch
 )
 
+find_program(XMLLINT_PATH NAMES xmllint PATHS "${CURRENT_HOST_INSTALLED_DIR}/tools/libxml2")
+string(REGEX REPLACE "/[^/]+$" "" XMLLINT_DIR "${XMLLINT_PATH}")
+file(TO_NATIVE_PATH "${XMLLINT_DIR}" XMLLINT_DIR_NATIVE)
+message(STATUS "Using xmlling at: ${XMLLINT_PATH}")
+vcpkg_add_to_path("${XMLLINT_DIR_NATIVE}")
```
Comment (dg0yt) [inline ports/gtk/portfile.cmake:18]: Why not just
~~~cmake
vcpkg_add_to_path("{CURRENT_HOST_INSTALLED_DIR}/tools/libxml2") # for xmllint
~~~

File: ports/gtk/vcpkg.json [hunk 2]
```diff
@@ -38,6 +38,12 @@
     },
     "graphene",
     "libepoxy",
+    {
+      "name": "libxml2",
+      "features": [
+        "tools"
+      ]
+    },
```
Comment (dg0yt) [inline ports/gtk/vcpkg.json:46]: I guess this is meant to be `"host": true`.

Comment (FrankXie05) [reply ports/gtk/vcpkg.json:46]: This feature is necessary.

Comment (dg0yt) [reply ports/gtk/vcpkg.json:46]: Grrr. I didn't mean to remove the feature. I mean that you don't get the required host tool if you don't ask for a host tool.

**Pull request closed without merging**